### DISCUSSION TITLE
Mapping of address based on lat / lon is inconsistent since Feb 10, 2021

### DISCUSSION THREAD

**Discussion by akragel** [osm-search/Nominatim#2206] [Q&A]
Hi,
Thank you for great product. We have been using it for a long time and like it.

I need your help to better understand what can be done.

I noticed that Nominatim recently started having some mapping issue: mapping dropped by average 15% compare before Feb 10th, 2021 (85% now vs 99% old). 

I have checked on my end and all seems fine (code was not touched since Oct 2019). 
Could you please let me know if you recently touched, improved or redesign mapping (lon/lat based) functionality? 

PS. I have checked and only changes that I see are the installable version of Nominatim, which based on the dates might be related. 
Would be happy to work with you to understand what happening or what I can do on my end to improve mapping! 

Thank you for you help.

**Comment by mtmail** [comment]
Some changes are related to importing or indexing and only have an effect once a place in OSM data gets updates, so some effects take weeks to be visible. Then there's also OSM data changes from tiny to large (restructuring whole states, possibly broken data, or fixed data with side-effects). Can you list a couple of the queries you're running and which areas of the world they cover? Ideally full URLs. What does "mapping dropped" mean in your context?

**Reply by akragel** [reply]
I am running something like that 
` http://XXX-nominatim-XXX/nominatim/reverse.php?format=json&lat=28.364727&lon=-81.614519&zoom=18&accept-language=en`

Dropped means that Nominatim seems not able to provide information for 15% of lat and lon that we set to it.

**Reply by mtmail** [reply]
Can you check if in February the TIGER data (US Census 2020) on that machine was updated? https://github.com/osm-search/Nominatim/pull/2159 pointed to a new file, but the file was faulty. Latitude and Longitude were mixed up. Fixed a week later https://github.com/osm-search/Nominatim/pull/2166 Run `select count(*) from location_property_tiger;`, if it returns less than 34 million you should be fine, if 34 million then it might be that file.

**Reply by akragel** [reply]
@mtmail Thank you, let me check on that.

**Reply by akragel** [reply]
@mtmail Seems like our end-point was restarted at some point and DB folder was not found (postgres is not running). 

I wonder:
1.  How I could check TIGER data (US Census 2020) without running SQL query? is there something like a file? 
2. Do you have a suggestion how to bring it up without updating whole project? 
2. And maybe you guys could please help me understand why is your cod works even without DB running?

Thank you very much for your help!

**Reply by mtmail** [reply]
Nominatim doesn't work without database connection, it will just print errors. You need to find out what your program or website was querying, it sounds like it wasn't the Nominatim installation. If it was something different, a fallback provider maybe, that is a good explanation why results have been different.

Importing TIGER data is explained at https://nominatim.org/release-docs/latest/admin/Import/#installing-tiger-housenumber-data-for-the-us (assuming you use Nominatim version 3.6, you can change the version on the top-left of the page). It can be run at any time, even multiple times and doesn't require reimporting all OpenStreetMap data. It takes 1-2 hours to run and is manual started. If you think the server ran for months without somebody actively maintaining it, then I highly doubt somebody ran the script.
